### ISSUE TITLE
Spread whalesong

### ISSUE THREAD

**Issue by alfred82santa** [alfred82santa/whalesong#72]
We need users who ask for new features. We have to improve communication:

* [ ] Whalesong logo
* [ ] Whalesong webpage

**Comment by kuarad** [comment]
Yes, this a great project, i have been testing for about 3 months, with no issues for my purposes, and i have new features sugestions, i have made some modifications on my environment, and i'll post here soon as possible

**Comment by vasani-arpit** [comment]
@alfred82santa I can create logo. I have a few questions/suggestions on that. How can I send those over to you? Your email/skype ?

**Comment by alfred82santa** [comment]
@vasani-arpit What kind of questions/suggestions? Cannot we talk in a public issue? Maybe others want to collaborate.

**Comment by vasani-arpit** [comment]
The logo, Should it have resemblance of Whatsapp? or It should be like whale like docker?

**Comment by alfred82santa** [comment]
Well, I would like avoid any reference to Whatsapp, chats, menssegers or anythink like that. It's true, currently whalesong is just a driver for WhatsAppWeb, but the idea was to create a framework to be able to control/manage js applications. It didn't matter what kind of apps.

**Comment by vasani-arpit** [comment]
### Check this out
![logo2](https://user-images.githubusercontent.com/6497827/57604554-12049800-7582-11e9-818e-41bb192328bd.jpg)
![logo3](https://user-images.githubusercontent.com/6497827/57604556-12049800-7582-11e9-8815-352e16fef3cd.jpg)
![logo1](https://user-images.githubusercontent.com/6497827/57604557-129d2e80-7582-11e9-945a-c436bfcab07b.jpg)

**Comment by alfred82santa** [comment]
I like the first one and the last one... but the last one looks like kuebernetes google service. Looking at first one, what do you think about change perspective to isometric (like last one) and remove extra stuff inside cube?

**Comment by alfred82santa** [comment]
Last one looks like this one:
![imatge](https://user-images.githubusercontent.com/574291/57913753-965e6000-788d-11e9-8316-44c4654e41d7.png)

**Comment by vasani-arpit** [comment]
> I like the first one and the last one... but the last one looks like kuebernetes google service. Looking at first one, what do you think about change perspective to isometric (like last one) and remove extra stuff inside cube?

Let me try that.

**Comment by vasani-arpit** [comment]
Note sure I got all. But how about this?

![logo2](https://user-images.githubusercontent.com/6497827/57917856-38447380-78b3-11e9-820c-0553b177d7f4.jpg)

**Comment by alfred82santa** [comment]
I was thinking with something like that but with an isometric cube

**Comment by vasani-arpit** [comment]
![378  Converted](https://user-images.githubusercontent.com/6497827/57965975-77350080-7969-11e9-9505-48b7bf6c5e7a.jpg)

![945  Converted](https://user-images.githubusercontent.com/6497827/57965980-8451ef80-7969-11e9-89bb-1d8af515926f.jpg)

**Comment by vasani-arpit** [comment]
@alfred82santa have you checked this?

**Comment by alfred82santa** [comment]
well, I think they are too much complex. I was thinking in something more symbolic. It must be valid for logo on website, but also valid for icon on other projects, I mean, other project could be use it like: powered by whalesong.

**Comment by vasani-arpit** [comment]
Well that was the end of my Creativity in Design. Ever since I have moved to Development I find my self doing designing less and less everyday. Sorry !!
